### DISCUSSION TITLE
(libreoffice-help) package "LibreOffice Help Pack 5.2.0" outdated by far, seems unmaintained for long time

### DISCUSSION THREAD

**Discussion by zWhdmB5T** [chocolatey-community/chocolatey-packages#2311] [General]
# [https://github.com/chocolatey-community/chocolatey-packages/issues/2310](https://github.com/chocolatey-community/chocolatey-packages/issues/2310)

### New Software Version

7.5.6

### Download location

https://download.documentfoundation.org/libreoffice/stable/7.5.6/win/x86_64/

### Package Page

https://github.com/chocolatey-community/chocolatey-packages/tree/master/manual/libreoffice-help

**Comment by Starmania** [comment]
Hmmm why this is not an AU package...

**Comment by zWhdmB5T** [comment]
Well, the main path for downloading the language pack is

https://download.documentfoundation.org/libreoffice/stable/7.5.6/win/x86_64/

and a specific language file is

https://download.documentfoundation.org/libreoffice/stable/7.5.6/win/x86_64/LibreOffice_7.5.6_Win_x86-64_helppack_en-US.msi

for (US) English (International) for example

or

https://download.documentfoundation.org/libreoffice/stable/7.5.6/win/x86_64/LibreOffice_7.5.6_Win_x86-64_helppack_de.msi

for German language for example.

The version code (i.e. 7.5.6) inside the link is quite obvious and is relating to 

https://community.chocolatey.org/packages/libreoffice-still

package (LibreOffice Still, in this case 7.5.6 currently).

So, I guess this could be turned into an AU package. Anyone?
